src/error/generated.ts

```diff
@@ -446,6 +442,92 @@ export namespace AuthorizationErrors {
     }
   }
 
+  /**
+   * @public
+   * AuthorizationErrors.CallMessageEventTypeInvalidError error.
+   * Error code `31210`.
+   */
+  export class CallMessageEventTypeInvalidError extends TwilioError {
+    /**
+     * The Call Message Event Type is invalid and is not understood by Twilio Voice.
```

Comment:
Are these comments necessary if the syntax is the exact same as the content below it?

Reply:
I think it's beneficial to have the comments. For starters, it's part of the generation script, so we would have to change all the errors and the generation script, and second, it puts the description into the API docs, otherwise it would just say "string".